docs/content/testing/running-tests.md

```diff
@@ -0,0 +1,62 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Running tests
+
+Run the following commands in your terminal from the root of the repository.
+
+For the first run, you need these commands:
```

Comment:
(1) What does 'the first run' mean? The first time I run any of the tests? Can you be specific?
(2) Do I need these commands at any other time? If not, then tell me I only need to use them once. Or do I need to run these commands before each test? And only once?
(3) What does 'need' mean? I need to run these commands? Or I need to have them ready to run? Can you be explicit?

Reply:
Rephrasing this part too